src/MCPClient/debian/copyright

```diff
@@ -1,6 +1,6 @@
 This work was packaged for Ubuntu by:
 
-    Justin Simpson <jsimpson@artefactual.com>
+    Artefactual Systems Inc. <sysadmin@artefactual.com>
```

Comment:
Do we want to use ops@artefactual.com (as in the control file) or sysadmin@artefactual.com (as here).  Are they supposed to be the same?